### DISCUSSION TITLE
Problem with build, Can't resolve 'react-router-dom'

### DISCUSSION THREAD

**Discussion by DanielCok17** [vercel/community#1629] [Help]
### Summary

Guys, have you ever encountered this mistake when building? I'm using react-router-dom to navigate. I tried to solve this problem by adding a vercel file. json to the root directory from with overwriting the source and destination directories, but it didn't solve my problem. Pls, any advice? Thanks

**Comment by amyegan** [comment]
Hi Daniel! Are you able to share a minimal reproduction (or your repo if it is public) so I can get a better look at what is going wrong?
